src/main/java/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,50 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class DeleteHistories {
+    private List<DeleteHistory> deleteHistories = new ArrayList<>();
+
+    public DeleteHistories() {
+    }
+
+    public DeleteHistories(DeleteHistory deleteHistory) {
+        this.add(deleteHistory);
+    }
+
+    public DeleteHistories(List<DeleteHistory> deleteHistories) {
+        this.deleteHistories.addAll(deleteHistories);
```

Comment:
```suggestion
        this.deleteHistories = deleteHistories;
```

위 코드가 더 깔끔할것 같아요!
개인적으로는 생성자에 `validate` 제외한 코드는 없는게 더 좋다라는 관점이에요
생성자는 순수한 그대로 작동되어야 안정된다는 생각때문이에요.

Reply:
생성자는 `validate` 제외한 코드가 없는게 더 좋다  
왜냐하면 생성자는 순수한 그대로 작동되어야 안정된다

공감가네요! 조언 리뷰 감사합니다!